plugin/src/main/scala/Giter8Plugin.scala

```diff
@@ -135,13 +135,24 @@ object Giter8Plugin extends sbt.AutoPlugin {
         val dir     = (sourceDirectory in Test).value
         val metadir = (baseDirectory in LocalRootProject).value / "project"
         val file0   = dir / "g8" / "test"
+
+        // we should only use file0 if its an exisiting file
+        // otherwise, we fallback to test.script
+        val defaultTestScript = 
+          if (file0.exists && file0.isFile) file0
+          else dir / "g8" / "test.script"
+
         val files = List(file0,
+                         dir / "g8" / "test.script",
                          dir / "g8" / "giter8.test",
                          dir / "g8" / "g8.test",
```

Comment:
I don't know how "giter8.test" and "g8.test" could possibly be used. If we add such a file, for instace, giter8 will pick it, but it won't work with `sbt-scripted` because it only accepts "test" or "test.script".

Reply:
oh, now I see why "giter8.test" and "g8.test" can work. The user can define the test in such a file, but then when it gets run, the content of the file is copied to `out / "test"` (as seen [here](https://github.com/foundweekends/giter8/pull/408/files#diff-5276071456623889c8874e1d06555f46L126)). 

In the end, the test script is always copied to a file called "test". The 'fix' I did [here](https://github.com/foundweekends/giter8/pull/408/files#diff-5276071456623889c8874e1d06555f46R126) make it impossible to use "giter8.test" or "g8.test", because it makes the final script to be named as such which sbt-scripted won't recognize.